src/main/java/com/capitalone/dashboard/collector/DefaultArtifactoryClient.java

```diff
@@ -30,6 +30,7 @@
 import org.springframework.web.client.RestOperations;
 
 import java.nio.charset.StandardCharsets;
+import java.sql.Time;

```

Comment:
do we need sql time?

Reply:
looks like it's not used, so I will delete it